### PR TITLE
Shooter sim

### PR DESCRIPTION
**Pull request by YJ-7274**
Managed to integrate turnandshoot into the simulator, also edited kvVoltSecondsPerRadian in Constants in order to make aiming more accurate.

### CODE REVIEW

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -26,6 +27,12 @@
 
   private AutoCommandInterface m_autonomousCommand;
   private RobotContainer m_robotContainer;
+  // private DriveTrain driveTrain;
+   private Carousel carousel;
```
Comment (LQ1234) [inline src/main/java/frc/robot/Robot.java]:
```suggestion
  private Carousel carousel;
```

**Comment by YJ-7274** [comment]
larry plz approve -_-

**Review by LQ1234 — APPROVED** [review]
Ok yes YJ LGTM except some formatting

**Comment by LQ1234** [comment]
Actually wait a sec
You added
```
  // private DriveTrain driveTrain;
  private Carousel carousel;
  // private Intake intake;
  // private DriveCommand driveCommand;
  // private Shooter shooter;
  // private Climber climber;
```
to Robot.java
but there are already the same objects in RobotContainer.java
I'm not too familiar with this organization but is this correct?

**Comment by LQ1234** [comment]
@YJ-7274 why did you add this?

**Comment by coachFitz0807** [comment]
I fixed it. The variable carousel wasn't being used anyway. That might have been left over from the merge of the Vision subsystem.

**Pull request merged**